<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPhraseDialog</class>
 <widget class="QDialog" name="AddPhraseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Phrase</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="englishPhraseLabel">
       <property name="text">
        <string>&amp;English phrase:</string>
       </property>
       <property name="buddy">
        <cstring>englishPhraseLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="englishPhraseLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="russianPhraseLabel">
       <property name="text">
        <string>&amp;Russian phrase:</string>
       </property>
       <property name="buddy">
        <cstring>russianPhraseLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="russianPhraseLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="imageGroupBox">
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ITS.qrc">:/Images/StandardImage.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring></cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadImagePushButton">
        <property name="text">
         <string>&amp;Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ITS.qrc"/>
 </resources>
 <connections/>
</ui>
